<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsMain</class>
 <widget class="QWidget" name="SettingsMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: rgb(42, 42, 42);*/

QLabel
{
	color: rgb(216, 216, 216);
}

QPushButton 
{	
	color: rgb(216, 216, 216);
	background-color: rgb(0, 0, 0);
	border-radius: 10px;
}

QPushButton:hover 
{
	background-color: rgb(35, 11, 63);
}

QPushButton:disabled
{
	color: rgb(73, 73, 73);
}

QPushButton:pressed 
{
	background-color: rgb(82, 26, 149);
}

QTableWidget
{
	background-color: rgb(0, 0, 0);
}

QTableWidget::item::selected
{
	background-color: rgb(0, 58, 201);
}

QHeaderView::section
{
	background-color: rgb(20, 6, 36);
}

QHeaderView
{
	background-color: rgb(30, 30, 30);
}</string>
  </property>
  <widget class="QWidget" name="Titlebar_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background-color: rgb(20, 6, 36);*/
background-color: rgb(27, 27, 27);</string>
   </property>
   <widget class="QLabel" name="Titlebar_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PlayList_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>   PlayList Editor</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/playlist.png</normaloff>:/icons/icons/playlist.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Close_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>  Close     </string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/white-left-arrow.png</normaloff>:/icons/icons/white-left-arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Library_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>   Library Editor</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/library.png</normaloff>:/icons/icons/library.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="HotKeys_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>   Hot Keys</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/keyboard.png</normaloff>:/icons/icons/keyboard.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="background_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>801</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.369, y1:0.182, x2:1, y2:1, stop:0 rgba(57, 9, 126, 255), stop:1 rgba(129, 18, 129, 255));</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenMediaFile_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>   Open Media File</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/videofile.png</normaloff>:/icons/icons/videofile.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenPlayList_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>   Open PlayList</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/playlists.png</normaloff>:/icons/icons/playlists.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Favorites_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>   Favorites Editor</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/star-white.png</normaloff>:/icons/icons/star-white.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="PlayerSettings_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>   Player Settings</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/settings.png</normaloff>:/icons/icons/settings.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="About_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>310</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>   About</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/white-information.png</normaloff>:/icons/icons/white-information.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <zorder>background_widget</zorder>
  <zorder>Titlebar_widget</zorder>
  <zorder>PlayList_button</zorder>
  <zorder>Close_button</zorder>
  <zorder>Library_button</zorder>
  <zorder>HotKeys_button</zorder>
  <zorder>OpenMediaFile_button</zorder>
  <zorder>OpenPlayList_button</zorder>
  <zorder>Favorites_button</zorder>
  <zorder>PlayerSettings_button</zorder>
  <zorder>About_button</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
